lib/blockscore/connection.rb

```diff
@@ -7,60 +7,51 @@ module Connection
     USER_AGENT    = "blockscore-ruby/#{BlockScore::VERSION} (#{REPO})".freeze
     CONTENT_TYPE  = 'application/json'.freeze
     API_URL       = URI.parse('https://api.blockscore.com').freeze
+    HEADERS       = {
+      'Accept'       => ACCEPT_HEADER,
+      'User-Agent'   => USER_AGENT,
+      'Content-Type' => CONTENT_TYPE
+    }.freeze
 
-    def get(path, params)
-      request(:get, path, params)
+    def delete(path, _)
```

Comment:
Or should it accept `params` and then do `encode_path_params(path, params)` like `#get`?